### PR TITLE
[Docs] Add documentation for kubectl and Helm Kanvas Snapshot extensions

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

- This PR adds new pages for forthcoming extensions.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c255b8800bf16ce26d71f4da93ab845328b71f18 at: https://67660f1344e08b9233228092--meshery-docs-preview.netlify.app

**Review by marblom007 — APPROVED** [review]
looking good here, I think. That's a lot of snapshotting. Different types, though, I get it.

File: docs/pages/extensions/helm-kanvas-snapshot copy/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+category: kanvas
+---
+
+# <img style="height: 4rem; width: 4rem;" src="{{site.baseurl}}/assets/img/integrations/helm-controller/components/helm-chart/icons/color/helm-chart-color.svg" /> Helm Kanvas Snapshot
+
+## Extension Overview
+
+The **Kanvas Snapshot Helm Plugin** allows users to generate a visual snapshot of their Helm charts directly from the command line. It simplifies the process of creating Meshery Snapshots, providing a visual representation of packaged Helm charts. This plugin integrates with Meshery Cloud and GitHub Actions to automate the workflow of snapshot creation, which is especially useful for Helm users who need to quickly visualize their chart configurations.
+
+Helm charts can be complex, especially when custom configurations are applied via `values.yaml` files. This Meshery extension bridges the gap between Helm chart configurations and their visual representation by converting Helm charts into **Kanvas Snapshots**. These snapshots can be received either via email or as a URL displayed directly in the terminal.
```
Comment (ritzorama) [inline docs/pages/extensions/helm-kanvas-snapshot copy/index.md:19]: This is true!

**Review by ritzorama — APPROVED** [review]

**Comment by leecalcote** [comment]
@theBeginner86 @Aijeyomah heads-up

**Pull request merged**